post.hbs

```diff
@@ -31,6 +21,19 @@
                             </span>
                         {{/foreach}}
                     </span>
+                    <span>
+                        {{reading_time}}
+                    </span>
+                </span>
+
+                <span class="flex flex-wrap content-center gap-4 justify-center"> 
+                {{#foreach tags}}
+                    <div class="flex-none text-base text-black hover:underline border-2 border-black hover:border-yellow-500 rounded-full px-3">
```

Comment:
It would be good if, on hover, the background darkens slightly. See https://material-ui.com/components/chips/ for an example.

Reply:
Done